### PR TITLE
Use annual land

### PR DESCRIPTION
**Pull request by kvcalvin**
- Switches initialization data to use a single year instead of a five year average (and updates test data accordingly)
- Calculates harvested area and uses that to compare to FAO
- Uses more historical prices in forming expectation

### CODE REVIEW

**Comment by rplzzz** [comment]
Anyone know why we're not getting comments in the gcamland slack channel?

**Comment by rplzzz** [comment]
What about now?

File: R/lagged_expectation.R [hunk 1]
```diff
@@ -65,15 +65,34 @@ LaggedExpectation_calcExpectedPrice <- function(aLandLeaf, aPeriod, aScenarioInf
...
+      while(i < prevYear) {
+        previousExpectation <- aScenarioInfo$mLaggedShareOld * previousExpectation +
+          (1 - aScenarioInfo$mLaggedShareOld) * price_table$price[price_table$year == i]
+        i <- i + timeStep
+      }
     }
+
+    # Calculate expected price
+    expectedPrice <- aScenarioInfo$mLaggedShareOld * previousExpectation +
+      (1 - aScenarioInfo$mLaggedShareOld) * newInformation
```
[the remark this answers is not in the document]
Comment (kvcalvin) [reply R/lagged_expectation.R]: I think the formula should be `y[i] = a*y[i-1] + (1-a)*x[i-1]`. You won't know `x[i]` until after you have made a decision using `y[i]`.

[the remark this answers is not in the document]
Comment (rplzzz) [reply R/lagged_expectation.R]: Well, this is what I was saying above.  The idea here isn't to have a literal procedure that reflects some calculation that an agent is making; rather, it is to have some abstract process that reflects the concept that "agents' estimates of prices lag somewhat behind the real thing", where "somewhat" is defined by an adjustable parameter.  The limiting behaviors are, when `a->1` agents never update their prices (note I got this backward in the comment above), and when `a->0` agents forecasts are bang-on every time (the model reduces to the perfect foresight model).

In reality agents are making their decision through some combination of historical prices, realtime forecasts that may or may not be any good, hunches based on years of experience, and heaven knows what else.  However, all we care about is how close that process gets them to the real thing (which is controlled by `a`).  We're not trying to simulate their actual decision process.

**Pull request merged**